src/frontend/screens/Settings/MapSettings/DeleteMapBottomSheet.tsx

```diff
@@ -0,0 +1,145 @@
+  },
+  deleteMap: {
+    id: "screens.Settings.MapSettings.DeleteMapBottomSheet.deleteMap",
+    defaultMessage: "Delete Map",
+  },
+  cancel: {
+    id: "screens.Settings.MapSettings.DeleteMapBottomSheet.cancel",
+    defaultMessage: "Cancel",
+  },
+});
+
+interface DeleteMapBottomSheetProps {
+  mapName: string;
+  mapId: string;
+  closeSheet: () => void;
+}
+
+export const DeleteMapBottomSheet = React.forwardRef<
+  BottomSheetMethods,
+  DeleteMapBottomSheetProps
+>(({ mapName, closeSheet, mapId }, sheetRef) => {
+  const [snapPoints, setSnapPoints] = React.useState<(number | string)[]>([
+    0,
+    "40%",
+  ]);
+
+  const { navigate } = useNavigationFromRoot();
+  const { formatMessage: t } = useIntl();
+  const { styleId, setStyleId } = useMapStyle();
+
+  function deleteMap() {
+    // Cannot delete Default Map
+    if (mapId === DEFAULT_MAP_ID) {
+      return;
+    }
+
+    // If user is deleting the map that is currently being used, we want to set the map to be the default map
+    if (styleId === mapId) {
+      setStyleId(DEFAULT_MAP_ID);
+    }
+
+    api.maps
+      .deleteStyle(mapId)
+      .then(() => navigate("BackgroundMaps"))
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
+  return (
+    <BottomSheet
+      ref={sheetRef}
+      snapPoints={snapPoints}
+      backdropComponent={BottomSheetBackdrop}
+      enableContentPanningGesture={false}
+      enableHandlePanningGesture={false}
+      handleHeight={0}
+      handleComponent={() => null}
+    >
+      <View
+        onLayout={e => {
+          const { height } = e.nativeEvent.layout;
+          setSnapPoints([0, height]);
+        }}
+        style={styles.btmSheetContainer}
+      >
+        <ErrorIcon style={{ position: "relative" }} size={90} color={RED} />
+
+        <Text style={{ fontSize: 24, textAlign: "center", margin: 10 }}>
+          <FormattedMessage
+            {...m.deleteMapMessage}
+            values={{ mapName: mapName }}
+          />
+        </Text>
+
+        <Text>
+          <FormattedMessage {...m.deleteMapWarning} />
+        </Text>
+
+        <Button
+          onPress={deleteMap}
+          fullWidth
+          color="dark"
+          style={{ backgroundColor: RED, marginTop: 30, marginBottom: 20 }}
+        >
+          <View
+            style={{
+              display: "flex",
+              flexDirection: "row",
+              justifyContent: "center",
+              alignItems: "center",
+            }}
+          >
+            <DeleteIcon color={WHITE} />
+            <Text style={[styles.deleteButton, { color: WHITE }]}>
+              {t(m.deleteMap)}
+            </Text>
+          </View>
+        </Button>
+        <Button onPress={closeSheet} fullWidth variant="outlined">
+          <Text style={[styles.deleteButton, { color: MAPEO_BLUE }]}>
+            {t(m.cancel)}
+          </Text>
+        </Button>
+      </View>
+    </BottomSheet>
+  );
+});
+
+const styles = StyleSheet.create({
```

Comment:
Maybe more of a follow-up but we could potentially extract and update https://github.com/digidem/mapeo-mobile/blob/develop/src/frontend/sharedComponents/BottomSheetModal/Content.tsx as a shared component for any bottom sheet content we create and use it for situations like this. otherwise it's more prone to visual inconsistency with the [design system](https://www.notion.so/digidem/d2b2903e77fc4a8cae6d6a8467e25b59?v=def381515abe45578065598c05682c75&p=f28600ee2800443a8bf0980e011933ba&pm=c)

based on the screenshot you provided, the amount of space below the bottom most button is too large

Reply:
have some of this done in https://github.com/digidem/mapeo-mobile/pull/1031 so feel free to ignore this, assuming the visual issues were resolved